viewer/settings.py

```diff
@@ -134,3 +141,36 @@
     RAVEN_CONFIG = {
         'dsn': SENTRY_DSN,
     }
+
+LOGGING = {
+    'version': 1,
+    'disable_existing_loggers': False,
+    'formatters': {
+        'json': {
+            '()': 'dockerflow.logging.JsonLogFormatter',
+            'logger_name': 'experiments-viewer',
+        },
+        'verbose': {
+            'format': '%(levelname)s %(asctime)s %(name)s %(message)s',
+        },
+    },
+    'handlers': {
+        'console': {
+            'level': 'DEBUG',
+            'class': 'logging.StreamHandler',
+            'formatter': 'verbose',  # TODO: Use 'json' for Dockerflow.
```

Comment:
I would recommend introducing something akin to what we do with atmo, separating Django settings per environments (dev, stage, prod) and then have a boolean setting that switches this formatter between `verbose` and `json`. You probably have seen it in the atmo code. If you don't feel like django-configurations is your cup of tea, I can recommend [django-environ](http://django-environ.readthedocs.io/).